tools/graphovl.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env python3
```

Comment:
This should really be a subrepo rather than just including all of the files in a copy/paste manner.

Reply:
Okey. 
What is prefered? subrepo or submodule?

Reply:
`graphovl.py` is now a subrepo.